### DISCUSSION TITLE
Questions about expanding and contracting genes on tree branches and species nodes

### DISCUSSION THREAD

**Discussion by wen10-git** [hahnlab/CAFE5#159] [Q&A] [answered]
Hello @benfulton @xylebori @fkmendes 

This is a great tool and very useful!

But the problem I have now is how to interpret the expanding/shrinking genes on the "branches"; and on the species nodes. As you can see, the values on the branches are very small, but the ones on the species nodes are very large.

Please allow me to describe my research idea. I would like to understand what genes are expanded/contracted at some key evolutionary nodes, and thus relate them to their environment. But my confusion about this software right now centers on the values; the values strike me as a bit strange. For example, I'd like to describe some nodes giving birth to a certain genus, but the number of genes in the node corresponding to the genus may not change that much. However, there are large values on species nodes. So I am not particularly sure how to explain the problem from a biological point of view. I think it would be good research if I could rationalize that type of problem from a biological evolutionary perspective!

Thanks!

![image](https://github.com/hahnlab/CAFE5/assets/63379795/5cd175f5-77de-4d91-9888-3d2f678ad500)

**Comment by benfulton** [comment, marked as answer]
I asked Dr. Hahn's opinion on this, and he suggested a few things you might be seeing:

1. The tree is very deep, and there are whole genome duplications (WGD's) on many of the tip branches
2. The tip branches for most of the tree are the longest, so they will have the most changes
3. We probably have slightly more resolution toward the present, which might slightly inflate things
4. On the bottom of the tree it looks like there will be lots of gene tree discordance--this will also inflate changes a lot

Points 2 and 4 are probably the biggest effects.
